bindings/wasm/src/account/types/key_location.rs

```diff
@@ -40,6 +43,12 @@ impl WasmKeyLocation {
   pub fn generation(&self) -> WasmGeneration {
     self.0.generation().into()
   }
+
+  /// Serializes `Signature` as a JSON object.
+  #[wasm_bindgen(js_name = toJSON)]
+  pub fn to_json(&self) -> Result<JsValue> {
+    JsValue::from_serde(&self.0).wasm_result()
+  }
```

Comment:
Ideally we should have both complementary `fromJSON` and `toJSON` implementations.